src/theme/themeStyles.ts

```diff
@@ -1,55 +1,11 @@
-import { createTheme } from '@material-ui/core';
+import { createTheme } from '@mui/material';
+
 import theme from './themeProperties';
 
 const materialTheme = createTheme({
-  overrides: {
```

Comment:
@Shreyaschorge Didn't follow this. Are these overrides not needed any more?

Reply:
The way we used to override the theme has been changed
https://mui.com/material-ui/guides/migration-v4/#theme-structure

The following components were from [MuiPickers](https://material-ui-pickers.dev/) [npm library](https://www.npmjs.com/package/@material-ui/pickers). 

Earlier Overrides for date-pickers were done like this https://material-ui-pickers.dev/guides/css-overrides#override-example

Now since MUI authors moved date-pickers into another library
For now there is no standard way (Writing styles in overrides) to Style these Components.